src/Impl/AmqpBind.php

```diff
@@ -39,13 +39,13 @@ class AmqpBind implements InteropAmqpBind
      * @param int             $flags
      * @param array           $arguments
      */
-    public function __construct(AmqpDestination $target, AmqpDestination $source, $routingKey = null, $flags = self::FLAG_NOPARAM, $arguments = [])
+    public function __construct(AmqpDestination $target, AmqpDestination $source, $routingKey = null, $flags = self::FLAG_NOPARAM, array $arguments = [])
     {
-        $this->target = $target;
-        $this->source = $source;
+        $this->target     = $target;
```

Comment:
While PSRs are lenient to any or none indentation, it looks more readable, sure, if you abuse, you'll get messed diffs, but anyway, I'll drop formatting commit to stay on topic.